aoa_counter/src/settings.ts

```diff
@@ -0,0 +1,36 @@
+import { z } from 'zod';
+import * as fs from 'fs';
+import * as path from 'path';
+
+const connectionParams = {
+    protocol: z.union([z.literal('http'), z.literal('https'), z.literal('https_insecure')]),
+    ip: z.string(),
+    port: z.number(),
+    user: z.string(),
+    pass: z.string(),
+};
+const cameraSchema = z.object({
+    ...connectionParams,
+    serviceID: z.number(),
+    fieldName: z.string(),
+});
+const aoaSchema = z.object({
+    ...connectionParams,
+    updateFrequency: z.number(),
+    scenarioId: z.number(),
+    method: z.union([z.literal('getOccupancy'), z.literal('getAccumulatedCounts')]),
+});
+const settingsSchema = z.object({
+    camera: cameraSchema,
+    aoa: aoaSchema,
+});
+
```

Comment:
Neprijde mi idealni, ze mas schemata zkopirovane.. v ostatnich skriptech je schema jen symlink do react slozky aby bylo garantovane, ze pouzivaj vzdy stejne schema a myslim, ze to je lepsi reseni.